src/css/breakpoints.scss

```diff
@@ -0,0 +1,109 @@
+@import './variables.scss';
+@import './mixins.scss';
+
+@media (max-width: 450px) {
```

Comment:
Smart to make the breakpoint a separate file.  Would it be work breaking them into separate files for each break point?